clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/converters/AppengineAtomicOperationConverterHelper.groovy

```diff
@@ -40,7 +41,7 @@ class AppengineAtomicOperationConverterHelper {
       .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
       .convertValue(input, targetDescriptionType)
 
-    converted.credentials = credentials in AppengineNamedAccountCredentials ? credentials : null
```

Comment:
What on earth did `in` mean in this context?

Reply:
Great question. I am not a groovy specialist but `credentials in AppengineNamedAccountCredentials` mysteriously returns true.